static/bidder-info/imds.yaml

```diff
@@ -11,8 +11,10 @@ capabilities:
       - banner
       - video
 userSync:
-  # imds supports user syncing, but requires configuration by the host. contact this
-  # bidder directly at the email address in this file to ask about enabling user sync.
+  cookie-family-name: imds
```

Comment:
I believe `cookie-family-name` is what PBS-Java decided to call this value. In PBS-Go it is simply `key`. You don't need to specify the `key` in this file since it's the same as the default. You only need it for the synacormedia alias.

Reply:
Removed the `key` from imds yaml file.